### ISSUE TITLE
Antimalware Service Executable high CPU usage when running svn commit/update task by shortcut key

### ISSUE THREAD

**Issue by Monkey-D-Pixel** [Microsoft/vscode#63078]
- VSCode Version: 
Version: 1.29.0 (user setup)
Commit: 5f24c93878bd4bc645a4a17c620e2487b11005f9
Date: 2018-11-12T07:47:15.448Z
Electron: 2.0.12
Chrome: 61.0.3163.100
Node.js: 8.9.3
V8: 6.1.534.41
Architecture: x64

- OS Version: Windows 10 Professional 1809 (Build 17763.55)

Steps to Reproduce:

1. Use shortcut keys to run svn commit/update tasks.
2. Vscode gets slow, Windows task manager showing high CPU usage from "Antimalware Service Executable". After 10 or 20 secends, integrated terminal is expanded and the tasks finally get started and done.

It's interesting that the "ASE" does not react when do the commit/update by using TortoiseSVN client or by typing commands in the terminal.

This problem is since the 1.29.0 update.

Does this issue occur when all extensions are disabled?: Yes.

**Comment by dbaeumer** [comment]
@Monkey-D-Pixel is this happening every time. We see something comparable with gulp but only when starting the task the first time.

**Comment by Monkey-D-Pixel** [comment]
> @Monkey-D-Pixel is this happening every time. We see something comparable with gulp but only when starting the task the first time.

yes, every time

**Comment by Monkey-D-Pixel** [comment]
might be a specific case of #63055, not just svn tasks

**Comment by dbaeumer** [comment]
Adding @alexr00 since she is on #63055. How does the SVN task look like? Can you provide use with a GitHub repository we can clone that demos this?

**Comment by dbaeumer** [comment]
One try. If you add the following settings

```json
	"npm.autoDetect": "off",
	"gulp.autoDetect": "off",
	"grunt.autoDetect": "off",
	"jake.autoDetect": "off"
```

to the workspace settings does it still reproduce ?

**Comment by Monkey-D-Pixel** [comment]
@dbaeumer just user defined configurations in task.json, like this:
`        {
            "taskName": "svn update",
            "type": "shell",
            "command": "svn update ${fileDirname} ",
            "group": {
                "kind": "svn",
                "isDefault": true
            }
        }`
It's still reproducing after adding those settings.

And just found more about this:
If I ctrl+p, select "Run task" in the command palette, **MOSTLY** "ASE" will also spike, while there is a loading indicator at the bottom of the palette until the tasks are loaded. Then I choose one, the task is invoked immediately. **SOMETIMES**, normally after several times of trying the same steps, the tasks are loaded very quickly, with "ASE" not spiking.
So I think there is something wrong during loading tasks.

**Comment by alexr00** [comment]
I can repro. It does look as though something is very slow during loading tasks.

**Comment by skarcha** [comment]
I think the sequence is: (slow down and DOES NOT work) -> (slow down and work) -> (slow down and DOES NOT work) -> (slow down and work) -> ...

**Comment by dbaeumer** [comment]
I forgot one task provider. Can you disable that one as well:

```
"typescript.tsc.autoDetect": "off"
```

**Comment by Monkey-D-Pixel** [comment]
@alexr00 Yes, slows down both times.
@dbaeumer Finally, NO MORE slow down with this line added. But why?

**Comment by dbaeumer** [comment]
@Monkey-D-Pixel TypeScript provides task to build typescript project automatically and this seems to gotten slower in 1.29.

**Comment by alexr00** [comment]
@mjbvz, any ideas on why the typescript task provider has gotten slower in 1.29?

**Comment by roblourens** [comment]
Does this involve searching? Does it do a `findFiles('**/tsconfig.json')` or something?

**Comment by Kcko** [comment]
> I forgot one task provider. Can you disable that one as well:
> 
> ```
> "typescript.tsc.autoDetect": "off"
> ```

Thx, it solved my problem #63026

**Comment by franklin-ross** [comment]
I believe this is the same issue as https://github.com/Microsoft/vscode/issues/63055.

**Comment by mjbvz** [comment]
Yes, same root cause as #63070. The tsc task detector searches the workspace for all `tsconfig.json` files.

**Comment by alexr00** [comment]
@roblourens for FYI.
